docs/source/intro/datasets.rst

```diff
@@ -44,47 +44,64 @@ For tabular data files, the software accepts a set of predetermined column names
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
 | Name     | Column names                                                                                            | Mandatory |
 +==========+=========================================================================================================+===========+
-| title    | title, primary_title                                                                                    | yes\*     |
+| ID       | record_id                                                                                               | no        |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
-| abstract | abstract, abstract note                                                                                 | yes\*     |
+| Title    | title, primary_title                                                                                    | yes\*     |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
-| keywords | keywords                                                                                                | no        |
+| Abstract | abstract, abstract note                                                                                 | yes\*     |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
-| authors  | authors, author names, first_authors                                                                    | no        |
+| Keywords | keywords                                                                                                | no        |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
-| doi      | doi                                                                                                     | no        |
+| Authors  | authors, author names, first_authors                                                                    | no        |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
-| included | final_included, label, label_included, included_label, included_final, included, included_flag, include | no        |
+| DOI      | doi                                                                                                     | no        |
++----------+---------------------------------------------------------------------------------------------------------+-----------+
+| Priors_1 | final_included, label, label_included, included_label, included_final, included, included_flag, include | no        |
```

Comment:
I think this only holds for ASReviewLAB, right? 
For simulation mode this column has a different purpose. Should that difference be implemented here? The name 'priors' is a bit confusing perhaps.

Reply:
Maybe "labels" is a more general term? 
If I remember correctly, in the future the export file get a column that is called 'priors' for indicating prior knowledge. That could lead to confusion.

Reply:
Added a more elaborative description of the dual-usage of this column. Maybe that helps.